src/Windows/R/Components/Impl/ContainerManager/Implementation/View/ContainerManagerControl.xaml

```diff
@@ -152,7 +153,15 @@
                              ToolTip="{x:Static components:Resources.ContainerManager_CreateLocalDocker_VersionTooltip}"
                              AutomationProperties.Name="{x:Static components:Resources.ContainerManager_CreateLocalDocker_VersionTooltip}" />
 
-                    <WrapPanel Grid.Row="5" Grid.Column="1" Grid.ColumnSpan="2" Margin="0,0,0,0">
+                    <imaging:CrispImage Grid.Row="4" Grid.Column="0" Style="{StaticResource ValidityIndicatorStyle}"
+                                        cwpf:AttachedProperties.IsValid="{Binding Path=IsPortAvailable}" />
+
+                    <TextBox Grid.Row="4" Grid.Column="1" Style="{StaticResource TextBoxSingleLineStyle}"
+                             Text="{Binding Path=Port, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                             ToolTip="{x:Static components:Resources.ContainerManager_CreateLocalDocker_PortTooltip}"
```

Comment:
You may want to add `rcontrols:Watermark.TextBoxHint="Local port"` or something like that